mne/io/eeglab/eeglab.py

```diff
@@ -60,8 +59,7 @@ def _to_loc(ll):
 
 
 def _get_info(eeg, montage, eog=()):
```

Comment:
I thought it just gets info? Not sure what you mean by "set info". It is slightly tricky to refactor it with the other IO functions. Because in `set` files the locations can be in either the `set` file or the `locs` file -- so you need some extra logic.